src/pages/home/owners/Songs.tsx

```diff
@@ -132,22 +132,6 @@ const Songs: FunctionComponent = () => {
                   ) }
                 </IconButton>
               ) }
-              <Stack
```

Comment:
Talked to product/design, they do not think we should include the price on the song of the artist page for MVP